test/test_fx.py

```diff
@@ -19,15 +19,25 @@
 import typing
 import unittest
 import warnings
-from collections import namedtuple
-from copy import deepcopy
 from math import sqrt
+from functorch.experimental import control_flow
```

Comment:
Please organize these import changes. This file is not covered by `PYFMT` linter.